packages/kbn-test/src/page_load_metrics/cli.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Url from 'url';
+
+import { run, createFlagError } from '@kbn/dev-utils';
+import { capturePageLoadMetrics } from './capture_page_load_metrics';
+
+export function runPageLoadMetricsCli() {
+  run(
+    async ({ flags, log }) => {
+      const kibanaUrl = flags['kibana-url'];
+      if (!kibanaUrl || typeof kibanaUrl !== 'string') {
+        throw createFlagError('Expect --kibana-url to be a string');
+      }
+
+      const parsedUrl = Url.parse(kibanaUrl);
+
+      const [username, password] = parsedUrl.auth
+        ? parsedUrl.auth.split(':')
+        : [flags.username, flags.password];
+
+      if (typeof username !== 'string' || typeof password !== 'string') {
+        throw createFlagError(
+          'Mising username and/or password, either specify in --kibana-url or pass --username and --password'
+        );
+      }
+
+      const headless = !flags.head;
```

Comment:
why don't use the same `headless` flag?

Reply:
The default option is headless: the true per design on Puppeteer side, I thought we might use `--head` over `--headless=false`